docs/build-tools/maven.md

```diff
@@ -27,7 +27,14 @@ The Bloop plugin for Maven doesn't need to be installed, you can run
 following command.
 
 ```bash
-$ mvn ch.epfl.scala:maven-bloop_2.12:@VERSION@:bloopInstall
+$ mvn ch.epfl.scala:maven-bloop_2.13:@VERSION@:bloopInstall
+```
+
+In some cases, when you are using generated sources in you project
+you might need to run:
+
+```bash
+$ mvn generate-sources ch.epfl.scala:maven-bloop_2.13:@VERSION@:bloopInstall
```

Comment:
Would it be possible that `bloopInstall` triggers `generate-sources` automatically?

Reply:
I tried to add it to a phase afterwards, but it still didn't run the generate-sources phase for the protobuf plugin. It might be due to the fact that we don't actually add the plugin to the build.

Reply:
Apparently, you can bind the plugin to a phase and it will be executed during that phase by default, but not invoke the phase when you run the plugin:

https://stackoverflow.com/questions/3934833/maven-mojo-plugin-how-to-define-phases-that-must-be-triggered-before-this-goal